packages/scene-composer/e2e/tests/local-scene/scene-composer--local-scene.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
+
+import { PlaywrightHelper } from '../utils/sceneHelpers';
+
+const localScene = '/iframe.html?args=&id=developer-scene-composer--local-scene';
```

Comment:
This code is not part of this CR, need proper rebase with @mumanity's changes so these don't show up in diff